task_line/src/main.cpp

```diff
@@ -91,10 +91,17 @@ int main(int argc, char** argv) {
         leftForwardPublisher.publish(pwm_forward_left);
 
         frontUpwardPublisher.publish(pwm_upward_front);
-        backUpwardPublisher.publish(pwm_upward_back); 
+        backUpwardPublisher.publish(pwm_upward_back);
+
+        ROS_INFO("PWM forward_right : %d", pwm_forward_right.data);
```

Comment:
It would be better if you add something to separate the messages in each iteration.